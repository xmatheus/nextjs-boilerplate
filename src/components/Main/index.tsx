import * as S from './styles'

const Main = () => (
  <S.Wrapper>
    <S.Logo
      src="/img/logo.svg"
      alt="Image de um átomo e React  Avançado escrito ao lado"
    ></S.Logo>
    <S.Title>React Avançadooo</S.Title>
    <S.Description>
      Typescript, Reactjs, Nextjs e Styled Components
    </S.Description>
    <S.Illustration
      src="/img/hero-illustration.svg"
      alt="desenvolvedor de frente para uma tela"
    ></S.Illustration>
  </S.Wrapper>
)

export default Main
