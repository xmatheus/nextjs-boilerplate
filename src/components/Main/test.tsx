import { render, screen } from '@testing-library/react'
import Main from '.'

describe('<Main />', () => {
  it('should render the Heading ', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /react avan√ßado/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
