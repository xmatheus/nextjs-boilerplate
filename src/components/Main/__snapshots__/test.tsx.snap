// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the Heading  1`] = `
<main>
  <h1>
    React Avan√ßado
  </h1>
</main>
`
